website_breadcrumb/views/templates.xml

```diff
@@ -6,6 +6,13 @@
 <data>
 
 <template id="breadcrumb" name="Breadcrumb snippet">
+    <t t-call="website_breadcrumb.breadcrumb-generator">
+        <t t-set='include_self' t-value='False'/>
+        <t t-set='revert_order' t-value='True'/>
+    </t>
+</template>
+
+<template id="breadcrumb-generator" name="Breadcrumb snippet generator">
```

Comment:
@Yajo IMO we can find pros and cons for the 2 POV. :+1: with the proposal of @JonathanNEMRY to disable the template of `website_breadcrumb` when we install `website_snippet_breadcrumb`. This proposal seems to be a good compromise to take care of your concern.